cla_backend/apps/cla_eventlog/management/commands/find_and_delete_old_cases.py

```diff
@@ -22,20 +22,23 @@ def handle(self, *args, **kwargs):
         instance = FindAndDeleteCasesUsingCreationTime()
         cases = instance.get_eligible_cases()
         django_command = sys.argv[1]
-        if args and args[0] == "delete":
-            if len(args) > 1 and args[1] == "no-input":
-                instance.run()
-            elif django_command == "test":
+
+        if django_command == "test":  # If command is run in test
+            if args and args[0] == "delete":
                 instance.run()
             else:
+                return cases
+
+        if django_command == "find_and_delete_old_cases":  # If command is run in terminal
+            if len(args) > 1 and args[1] == "no-input":
```

Comment:
I think I'd put this inside the `args[0] == "delete"` condition. I don't think I like the idea that "manage.py find_and_delete_old_cases whatsgoingon no-input" runs the deletion... 😅

Reply:
Done (nested branches ftw 😅 )